consular/tests/test_main.py

```diff
@@ -230,8 +257,25 @@ def test_sync_app_task(self):
         yield d
 
     @inlineCallbacks
+    def test_sync_app_labels(self):
+        app = {
+            'id': '/my-app',
+            'labels': {'foo': 'bar'}
+        }
+        d = self.consular.sync_app_labels(app)
+        consul_request = yield self.consul_requests.get()
+        self.assertEqual(consul_request['method'], 'PUT')
+        self.assertEqual(consul_request['path'], '/v1/kv/consular/my-app/foo')
+        self.assertEqual(consul_request['data'], 'bar')
```

Comment:
Yeah `foo` is the key, it's in the `'path'` I'm checking the line above.
The `consul_request` dict here is the mocked call to Consul, `data` here is the payload being sent in the HTTP request, which in this case is `bar`.